docs/api/lights/SpotLight.html

```diff
@@ -175,6 +175,13 @@ <h3>[property:Object3D target]</h3>
 			The spotlight will now track the target object.
 		</p>
 
+		<h3>[property:Boolean shadowAutoUpdate]</h3>
+		<p>
+			If set to *true*, the shadow camera will be automatically updated each frame to match the
+			geometry of the spotLight (a square frustum tightly fitting the cone of the spot light defined
```

Comment:
If set to *true*, the shadow camera frustum will be automatically sized to match the cone of the spotLight

Reply:
done